### DISCUSSION TITLE
2023-hepc/

### DISCUSSION THREAD

**Discussion by giscus** [koderpark/BlogComment#3] [Announcements]
# 2023-hepc/

1. 학교에 공문이 붙어있어 접할 기회가 있었고
 2. 마침 과외 등의 일정과 겹치는 일이 없었기에

가볍게 대회를 치고 왔습니다.

6문제 3시간 셋이었는데, 6문제 ABCDEF 중 A B C D 4문제를 해결하였고, 전체 등수는 32등이었습니다.

특별상 조건은 딱히 의도한 바 없어서 아마 못받을거같기는 합니다. 상 받을 생각 없이 출전했던 대회였고 생각보다

https://blog.koder.page/2023-hepc/

**Comment by Waterticket** [comment]
에리카 하이
